docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,1137 @@
+{
```

Comment:
<p>This is getting really complex. I have not reviewed this Pearson part yet.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
I couldn't find an easier way to solve it. Right after the first line, I even replaced one of parenthesis with its equivalent term `alpha` so I can use (3) and (4) to simplify it further before plugging the calculated values `\mu_prime` (8) and `\sigma_prime` (9) in `f(\mu_prime , \sigma_prime)`. 

I could have simply plugged in the values (8) and (9) to directly find `f_min` right from beginning of this cell; however, I think, that approach will make it more complex and harder to follow. 

---

I will address the previous comments. Do you want me to wait till you review this part as well?

Reply:
Yeah, why don't you take some time to see if it can be simplified before I go over it. No rush

Reply:
I think breaking it down to smaller parts and reminder reader of some previously-calculated equations can improve readability. I also think I skipped a few steps but I believe it can help reader to follow the process easier. So, if you do not mind, please ignore this part and check it out in next set of commits.

(I can also try to see if there is a different way to get to calculate `f_min`)